recipe/meta.yaml

```diff
@@ -1,24 +1,26 @@
-{% set version = "14.0.0.rc1" %}
+{% set version = "14.0.0.dev55c71c9eac9b" %}
```

Comment:
Maybe we can choose something that sorts more nicely?

Reply:
The output of `git describe` is sadly not the best for this commit: `llvmorg-14-init-12920-g55c71c9eac9b`, we can use the distance from the branching tag but that didn't seem to add any value imho.

Reply:
Can we represent the tensorflow version that this commit comes from in the version (or at least) build string?

E.g. `14.0.0.tf280.dev...`

Reply:
I would also be OK with `14.0.0.tf280_55c71c9eac9b`. Personally, I would like to have the git commit in the version to make its origin really obvious.